number theory/NumberTheory - GCD/GCD.py

```diff
@@ -0,0 +1,57 @@
+import tensorflow as tf
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.